assets/styles/_buttons.css

```diff
@@ -0,0 +1,38 @@
+.buttons {
+    display: flex;
+    width: 95%;
+    height: 10vh;
+    align-items: center;
+    margin: auto;
+    justify-content: space-between;
+  }
+  .buttons .btn {
+    outline: none;
+    border: none;
+    border-radius: 4px;
+    color: #fff;
+    background-color: #000;
+    border: 0.5px solid #fff;
+    font-size: 16px;
+    width: 28%;
+    height: 30px;
+    font-size: 14px;
+    margin: 0 3px;
```

Comment:
I'll suggest you also use more of relative units like `em` or `rem`.

Reply:
Oh okay I will start using those, my background is with px so that's why I use it a lot